avocado/utils/iso9660.py

```diff
@@ -301,7 +302,7 @@ def __init__(self, path):
         super(Iso9660Mount, self).__init__(path)
         self._mnt_dir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         process.run('mount -t iso9660 -v -o loop,ro %s %s' %
-                    (path, self.mnt_dir))
+                    (path, self.mnt_dir), sudo=True)
```

Comment:
Oh, now I see the results of the sudo "issue" I caught earlier.  So yeah, here's your immediate response to that request for change.